rust/src/bwrap.rs

```diff
@@ -372,9 +372,25 @@ impl Bubblewrap {
     }
 
     /// Set /var to be read-only, but with a transient writable /var/tmp
-    pub(crate) fn var_tmp_tmpfs(&mut self) {
+    /// and compat symlinks for scripts.
+    pub(crate) fn setup_compat_var(&mut self) -> CxxResult<()> {
+        let content_dirs = &["alternatives", "vagrant"];
+        for entry in content_dirs {
+            let varlib_path = format!("var/lib/{}", &entry);
```

Comment:
This is completely fine as is, but...perhaps cleaner to do e.g.:
`let varlib_path = Utf8Path::new("var/lib").join(entry);`
?

(We're not consistent on this, so feel free to just keep as is)